.github/workflows/release_check_version.yaml

```diff
@@ -34,7 +34,6 @@ jobs:
         run: |
           # Extract resonate version
           RESONATE_VERSION=$(./resonate -v | awk '{print $3}')
-          RESONATE_VERSION="v${RESONATE_VERSION}"
```

Comment:
This will solve the workflow, but I'm not aware of any other breaking change that this might introduce in the broader resonate landscape, if any. Because effectively `resonate --version` will always have a leading `v`, if move forward with this PR.